pom.xml

```diff
@@ -596,7 +596,7 @@
         <javax.activation.groupId>javax.activation</javax.activation.groupId>
         <javax.activation.artifactId>activation</javax.activation.artifactId>
         <jsieve.version>0.7</jsieve.version>
-        <spring.version>3.2.18.RELEASE</spring.version>
+        <spring.version>4.3.25.RELEASE</spring.version>
```

Comment:
just out of curiosity, can you check with last version while you are at it? (in an other PR if you want)

Reply:
Ok, will do in another PR.